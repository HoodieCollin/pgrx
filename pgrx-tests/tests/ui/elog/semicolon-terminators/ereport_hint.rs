use pgrx::prelude::*;

fn main() {}

#[pg_extern]
fn ereport_hint_must_terminate_with_semicolon() {
    ereport!(
        loglevel = PgLogLevel::LOG;
        errcode = PgSqlErrorCode::ERRCODE_SUCCESSFUL_COMPLETION;
        message = "...";
        detail = "detail";
        hint = "hint",
    );
}
