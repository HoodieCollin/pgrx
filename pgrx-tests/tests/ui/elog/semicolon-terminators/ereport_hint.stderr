error: The alternative invocation style requires component sections to be terminated with a semicolon.
  --> tests/ui/elog/semicolon-terminators/ereport_hint.rs:7:5
   |
7  | /     ereport!(
8  | |         loglevel = PgLogLevel::LOG;
9  | |         errcode = PgSqlErrorCode::ERRCODE_SUCCESSFUL_COMPLETION;
10 | |         message = "...";
11 | |         detail = "detail";
12 | |         hint = "hint",
13 | |     );
   | |_____^
   |
   = note: this error originates in the macro `ereport` (in Nightly builds, run with -Z macro-backtrace for more info)
